src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt

```diff
@@ -238,10 +238,10 @@ class EventController {
      * @see EventMiniGame.handleDonation
      */
     fun handleDonation(event: DonateEvent) {
-        if (event.amount == null) return // no clue why this would happen, but just in case
-        var value = event.amount.toDouble()
-        if (value < 0) return // no negative donations (don't think this is possible)
+        val value = event.amount.toDouble()
+        if (value <= 0) return // no negative donations (don't think this is possible)
```

Comment:
Please leave these change as is. [Discussion](https://discord.com/channels/397526357191557121/1315083687464800307/1316888061442785341)

```suggestion
        if (event.amount == null) return // no clue why this would happen, but just in case
        var value = event.amount.toDouble()
        if (value < 0) return // no negative donations (don't think this is possible)
```